owncloudApp/src/main/java/com/owncloud/android/providers/DocumentsStorageProvider.kt

```diff
@@ -444,6 +478,16 @@ class DocumentsStorageProvider : DocumentsProvider() {
 
     private fun toUri(documentId: String): Uri = Uri.parse(documentId)
 
+    private fun getFileByIdOrException(id: Long): OCFile =
```

Comment:
In which scenarios do you use `getFileById` instead of `getFileByIdOrException`? I mean, shouldn't we always take into account the exception?

Reply:
We are using it sometimes that it is not critical to get a nullable, and throwing an exception is not needed.